packages/core/src/server.ts

```diff
@@ -2004,6 +2008,17 @@ export class Server {
 
         const start = Date.now();
 
+        if (req.method.startsWith("GET:")) {
+            if (req.method === `GET:${this.config.healthcheckPath}`) {
```

Comment:
The alternative to this weird-looking thing would be to make it an `HTTPTransport` config setting instead, but that also felt wrong, as this is really a "server" thing, not a "transport" one.

Reply:
Actually, now that I think about it I think it **should** be a concern of the transport layer. In fact, this whole code block seems kind of pointless, since `HTTPReceiver` is already setting the `200` status code and the response body is irrelevant in this case, anyway. So yeah, let's make it a setting of `HTTPTransport`.